x/committee/types/committee.go

```diff
@@ -0,0 +1,140 @@
+package types
+
+import (
+	"fmt"
+	"time"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	govtypes "github.com/cosmos/cosmos-sdk/x/gov/types"
+	"gopkg.in/yaml.v2"
+)
+
+const MaxCommitteeDescriptionLength int = 512
+
+// ------------------------------------------
+//				Committees
+// ------------------------------------------
+
+// A Committee is a collection of addresses that are allowed to vote and enact any governance proposal that passes their permissions.
+type Committee struct {
+	ID               uint64           `json:"id" yaml:"id"`
+	Description      string           `json:"description" yaml:"description"`
+	Members          []sdk.AccAddress `json:"members" yaml:"members"`
```

Comment:
q: how are members added? through regular param change proposals?

Reply:
The module has a couple of gov proposals that create/update/delete committees. So its up to the normal gov to add members, change permissions etc.
In theory these proposals could be submitted to committees, allowing them to change themselves or each other, but it's disabled currently.